src/import-page/import-stepper/ImportStepper.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react';
+import {
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { Button } from '@edx/paragon';
+import { history } from '@edx/frontend-platform';
+import { RequestStatus } from '../../data/constants';
+import CourseStepper from '../../generic/course-stepper';
+import { IMPORT_STAGES } from '../data/constants';
+import { fetchImportStatus } from '../data/thunks';
+import {
+  getCurrentStage, getError, getFileName, getLoadingStatus, getProgress, getSavingStatus,
+} from '../data/selectors';
+import messages from './messages';
+
+const ImportStepper = ({ intl, courseId }) => {
+  const currentStage = useSelector(getCurrentStage);
+  const fileName = useSelector(getFileName);
+  const { hasError, message: errorMessage } = useSelector(getError);
+  const progress = useSelector(getProgress);
+  const dispatch = useDispatch();
+  const handleRedirectCourseOutline = () => history.push(`/course/${courseId}/outline`);
```

Comment:
lets put function under variable `declaration` and `useEffect`
it's a little unclear at this point.